src/main/java/com/google/edith/servlets/WeekInfo.java

```diff
@@ -0,0 +1,15 @@
+package com.google.edith.servlets;
+
+public class WeekInfo {
```

Comment:
Let's add a javadoc to this public class

Reply:
Added.